hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -55,6 +70,19 @@ It is essential for all clients writing to an S3Guard-enabled
 S3 Repository to use the feature. Clients reading the data may work directly
 with the S3A data, in which case the normal S3 consistency guarantees apply.
 
+## Moving off S3Guard
+
+How to move off S3Guard, given it is no longer needed.
+
+1. Unset the option `fs.s3a.metadatastore.impl` globally/for all buckets for which it
+   was selected.
```

Comment:
Would be nice to remind people to delete those S3Guard DynamoDB tables as provisioned IPOS will still cost money. I think we can also use the s3guard command line tool to destroy, e.g. `hadoop s3guard destroy s3a://guarded-table/`.

Reply:
once you remove the s3guard settings it becomes harder to work out the table to destroy, you need to use dynamodb URLs. Easiest just to use the console, especially as it lets you see if there are any orphan tables around.